pgjdbc/src/test/java/org/postgresql/test/ssl/LazyKeyManagerTest.java

```diff
@@ -21,6 +23,16 @@
 
 public class LazyKeyManagerTest {
 
+  @Test
+  public void testLoadP12Key() throws Exception {
+    String certdir = "../certdir/";
+    PKCS12KeyManager pkcs12KeyManager = new PKCS12KeyManager(certdir + "goodclient.p12", new TestCallbackHandler("sslpwd"));
+    PrivateKey pk = pkcs12KeyManager.getPrivateKey("user");
```

Comment:
It is unfortunate that `getPrivateKey` does not throw exceptions.
In other words, this method silently returns null, and the exception is not visible to the caller :(

Reply:
ya, very hard to debug